portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -1215,7 +1215,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-129398. Remember device with multiple auth."
+	@description = "This is a use case for LPS-129398. User can authorize multiple external applications which have the remember device option checked"
```

Comment:
Please end the description sentence with a dot "." to be more consistent.
cc: @ferenconodiD